### DISCUSSION TITLE
[MCC] Connection lost..... HELP Please :(

### DISCUSSION THREAD

**Discussion by Rembi96** [ORelio/Minecraft-Console-Client#2669] [Q&A]
Hey there, every few minutes and sometimes 1 or 2 hours i get the message [MCC] Connection lost..
why? Everything works perfect...  but this is bu**S**t :/ Anyone an idea?

**Comment by milutinke** [comment]
Use the [Auto Relog](https://mccteam.github.io/guide/chat-bots.html#auto-relog) chat bot.
